### PR TITLE
DYN-2328 Add test for geometry labeling

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

This PR is to add a test for Geometry Labeling. 
JIRA: https://jira.autodesk.com/browse/DYN-2328

The test checks that there is are no labels in the geometry when the DisplayLabels flag is set to false and asserts for the correct number of labels when the DisplayLabels flag is set to true. 

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@mjkkirschner @QilongTang @aparajit-pratap @zeusongit

### CODE REVIEW

**Comment by mjkkirschner** [comment]
@reddyashish nice - does this actually pass 😉 ? LGTM (if it does)

**Comment by mjkkirschner** [comment]
I'm also curious if had a change to look into the labels that appear when the individual preview bubble items are selected?

 Ie - when list 0 is selected - x labels should be visible, when list 1 is selected y labels should be visible.

File: src/VisualizationTests/HelixWatch3DViewModelTests.cs [hunk 1]
```diff
@@ -731,6 +731,41 @@ public void Display_ByGeometryColor_HasColoredMesh()
...
+            // By default the DisplayLabels for the code block node is set to false, 
+            // so the Model3DDictionary wouldn't have the geometry object corresponding to the Labels. 
+            var geometryHasLabels = helix.Model3DDictionary.ContainsKey(labelKey); 
+            Assert.IsFalse(geometryHasLabels);
+
+            // We set the DisplayLabels to true to view the Labels in the preview geometry.
+            codeBlockNodeModel.DisplayLabels = true;
+
+            helix = ViewModel.BackgroundPreviewViewModel as HelixWatch3DViewModel;
+            Model3DDictionary = helix.Model3DDictionary;
```
Comment (aparajit-pratap) [inline src/VisualizationTests/HelixWatch3DViewModelTests.cs]: Why do we need to assign these variables again? Would they not reflect the updated values after labels are turned on?

Comment (reddyashish) [reply src/VisualizationTests/HelixWatch3DViewModelTests.cs]: @aparajit-pratap yes, you are right. The helix object is updated when the labels are turned on and we do not need these assignments. Removing it.

**Comment by reddyashish** [comment]
@mjkkirschner Yes, this test passes. I am looking into labels when an individual item in the preview bubble is selected.

File: src/VisualizationTests/HelixWatch3DViewModelTests.cs [hunk 2]
```diff
@@ -258,7 +258,7 @@ public void Node_Labels_Render()
             OpenVisualizationTest("Labels.dyn");
 
             // check all the nodes and connectors are loaded
-            Assert.AreEqual(2, model.CurrentWorkspace.Nodes.Count());
+            Assert.AreEqual(3, model.CurrentWorkspace.Nodes.Count());
```
Comment (reddyashish) [inline src/VisualizationTests/HelixWatch3DViewModelTests.cs]: Since I added a watch node to this Labels.dyn, I had to update the node count in this test

**Comment by reddyashish** [comment]
@mjkkirschner updated the test with the case we discussed. 

This PR is ready to be merged once you guys take a look.

**Comment by mjkkirschner** [comment]
nice! LGTM

**Comment by reddyashish** [comment]
Merging this.

**Pull request merged**